src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/exchange_rates.html.twig

```diff
@@ -0,0 +1,45 @@
+{{ form_start(exchangeRatesForm, {'action': path('admin_currencies_update_exchange_rates') }) }}
+  <div class="card">
+    <h3 class="card-header">
+      {{ 'Exchange rate'|trans({}, 'Admin.International.Feature') }}
+    </h3>
+    <div class="card-block row">
+      <div class="card-text">
+        <div class="form-group row">
+          <label class="form-control-label" for="{{ exchangeRatesForm.live_exchange_rate.vars.id }}">
+            {{ 'Live exchange rates'|trans({}, 'Admin.International.Feature') }}
+          </label>
+          <div class="col-sm">
+            {{ form_widget(exchangeRatesForm.live_exchange_rate, {'attr': {'class': 'js-live-exchange-rate', 'data-url': path('admin_currencies_update_live_exchange_rates')}}) }}
+
+            {% if exchangeRatesForm.live_exchange_rate.vars.disabled %}
+                <small class="form-text">
+                  {{ 'Please install the %module_name% module before using this feature.'|trans({'%module_name%': 'cronjobs'}, 'Admin.International.Notification') }}
+                </small>
+              {% else %}
+                <small class="form-text js-exchange-rate-text-when-disabled {{ (exchangeRatesForm.live_exchange_rate.vars.value == 0) ? '' : 'd-none' }}">
+                  {{ 'The exchange rates are not automatically updated'|trans({}, 'Admin.International.Feature') }}
+                </small>
+                <small class="form-text js-exchange-rate-text-when-enabled {{ (exchangeRatesForm.live_exchange_rate.vars.value != 0) ? '' : 'd-none' }}">
+                  {{ 'The exchange rates are automatically updated'|trans({}, 'Admin.International.Feature') }}
+                </small>
+            {% endif %}
+          </div>
+        </div>
+        <div class="form-group row">
+          <label for="" class="form-control-label">
+            {{ 'Update exchange rates'|trans({}, 'Admin.International.Feature') }}
+          </label>
+          <div class="col-sm">
+            <button class="btn btn-primary">
+              {{ 'Update'|trans({}, 'Admin.Global') }}
```

Comment:
I think the word class here is the verb, not the noun - and in this case, I would rather localize this string in _Admin.Actions_. ;-)